docs/sources/configure-server/anonymous-usage-statistics-reporting.md

```diff
@@ -0,0 +1,60 @@
+---
+
+description: Learn about Grafana Pyroscope anonymous usage statistics reporting
+menuTitle: Anonymous usage statistics reporting
+title: About Grafana Pyroscope anonymous usage statistics reporting
+weight: 30
+---
+
+# About Grafana Pyroscope anonymous usage statistics reporting
+
+Grafana Pyroscope includes a system that optionally and anonymously reports non-sensitive, non-personally identifiable information about the running Pyroscope cluster to a remote statistics server.
+Pyroscope maintainers use this anonymous information to learn more about how the open source community runs Pyroscope and what the Pyroscope team should focus on when working on the next features and documentation improvements.
+
+The anonymous usage statistics reporting is **enabled by default**.
+You can opt-out setting the CLI flag `-usage-stats.enabled=false` or its respective YAML configuration option.
```

Comment:
I had to check this flag in Tempo. It's --reporting.enabled=false.

Reply:
Double checked this aspect, and yes the flag works as expected. 

(Log level debug shows when usage reports are sent).